readme.md

```diff
@@ -72,6 +72,7 @@ Some good apps written with Electron.
 - [electron-gh-release](https://github.com/jenslind/electron-gh-releases) - Auto-update by releasing on GitHub.
 - [electron-updater](https://github.com/evolvelabs/electron-updater) - Auto-updater leveraging npm to deploy updates.
 - [electron-download](https://github.com/maxogden/electron-download) - Download the Electron release zip from GitHub.
+- [electron-compile](https://github.com/paulcbetts/electron-compile) - Compile JS and CSS automatically in Electron apps.
```

Comment:
Confusing description as it's not really compiling JS/CSS, but rather precompiling down to those. Also `Electron` is moot as it's implied.